.pylintrc

```diff
@@ -88,6 +88,7 @@ disable=arguments-renamed,              # more readable and clear
         too-many-public-methods,        # too verbose
         too-many-statements,            # too verbose
         unnecessary-pass,               # allow for methods with just "pass", for clarity
+        not-context-manager,            # terra is not fully type-safe
```

Comment:
Qiskit is not fully typed, so trusting its type coming out of qiskit-land is sometimes too strong.